pkg/util/sys/device.go

```diff
@@ -263,7 +267,11 @@ func GetDeviceFromMountPoint(mountPoint string, executor exec.Executor) (string,
 	}
 
 	searchFor := fmt.Sprintf("on %s ", mountPoint)
-	device := Awk(Grep(output, searchFor), 1, " ")
```

Comment:
looks like `GetDeviceFromMountPoint` is also obsolete, just has a unit test referencing it

Reply:
Removed.